analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -1702,6 +1708,183 @@
     "    print(\"{0}: {1}\".format(subregion, percent[1]))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# geo_att Metric: Geography by attributes\n",
+    "<a id=\"geo_att\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
```

Comment:
the format of these notebooks is each metric has a "Setup" and an "Analyses" header... geo_att only has setup so far, no analyses